deps.bzl

```diff
@@ -2654,12 +2654,32 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
         sum = "h1:0Ut0rpeKwvIVbMQ1KbMBU4h6wxehBI535LK6Flheh8E=",
         version = "v4.17.2",
     )
+
+    go_repository(
+        name = "com_github_jackc_pgx_v5",
+        # TODO: Wait for https://github.com/bazelbuild/bazel-gazelle/issues/1465
+        # to be resolved, then remove this directive.
+        build_directives = [
+            "gazelle:build_tags unix",
+        ],
```

Comment:
can this be removed now?

Reply:
Hmm, turns out no, because the change isn't in a release yet. I'll change the comment to reflect this.